rasa/cli/arguments/train.py

```diff
@@ -88,7 +88,7 @@ def add_compare_params(
         "--percentages",
         nargs="*",
         type=int,
-        default=[0, 5, 25, 50, 70, 90, 95],
+        default=[25, 50, 75, 90],
```

Comment:
the 0 zero shouldn't hurt, right (cause that would just train with everything)? I think it's more the 95.